common/lib/capa/capa/inputtypes.py

```diff
@@ -85,6 +85,7 @@ def __init__(self, status, gettext_func=unicode):
         names = {
             'correct': _('correct'),
             'incorrect': _('incorrect'),
+            'partially-correct': _('partially-correct'),
```

Comment:
Huh. That's the first time I've ever gotten a useful response to that question in three years of working with edX; thank you very much.

It looks like it was David St. Germain a little over a year ago. Here's the commit comment:

"Convert CAPA inputtype statuses from strings to objects, in order to
simplify css class management and internationalization of status strings
(potentially) displayed to the user."

I also finally tracked down where it shows up: it's the status of the problem as read by screen readers. So it *is* user-facing, but you're also right that it *should* have a space in it. Will fix.